pkg/users/account-details.js

```diff
@@ -98,16 +98,27 @@ function get_expire(name) {
 
 export function AccountDetails({ accounts, groups, current_user, user, shells }) {
     const [expiration, setExpiration] = useState(null);
+    const [path, setPath] = useState("");
+
+    useEffect(() => {
+        const lastlog_version = cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
+        lastlog_version.then(() => {
+            setPath("/var/run/utmp");
+        }).catch(() => {
+            setPath("/var/lib/lastlog/lastlog2.db");
+        });
+    }, []);
+
     useEffect(() => {
         get_expire(user).then(setExpiration);
 
-        // Watch `/var/run/utmp` to register when user logs in or out
-        const handle = cockpit.file("/var/run/utmp", { superuser: "try", binary: true });
+        // Watch lastlog log to register when user logs in or out
+        const handle = cockpit.file(path, { superuser: "try", binary: true });
```

Comment:
This needs to check if the state is already initialized. Especially after testing for /var/lib/lastlog/lastlog2.db , there may be systems which have neither?